jobs/worker.go

```diff
@@ -57,8 +59,13 @@ func Run(shutdown chan struct{}) {
 		}
 	}()
 
+	//create struct for healthcheck struct and run the healthcheck thread
+	healthCheck := healthCheckStruct{timeStamp: time.Now(), timeChannel: make(chan time.Time, 10)}
```

Comment:
We should probably rename the struct to not include `struct` in the name. What do you think of `BackgroundWorkerHealthChecker`?